worker/runc/runc.go

```diff
@@ -67,6 +67,7 @@ func NewWorkerOpt(root string, snFactory SnapshotterFactory, rootless bool, proc
 		ApparmorProfile:     apparmorProfile,
 		TracingSocket:       traceSocket,
 		DefaultCgroupParent: defaultCgroupParent,
+		Hooks:               hooks,
```

Comment:
the `NewWorkerOpt` function might change overtime (in upstream commits), which will increase the odds of a conflict. It'll make maintaining this easier if we add an earthly-specific comment:

```suggestion
		Hooks:               hooks, //earthly-specific
```